### PR TITLE
DownloadPackageV1 log resolved latest version

### PR DESCRIPTION
**Pull request by embetten**
**Task name**: DownloadPackageV1

**Description**: log resolved latest version to console

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) #17621 

**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [X] Checked that applied changes work as expected

Example output with changes:
image

### CODE REVIEW

File: Tasks/DownloadPackageV1/main.ts [hunk 1]
```diff
@@ -74,8 +74,9 @@ async function main(): Promise<void> {
             tl.debug("Trying to resolve latest version for " + packageId);
             version = await p.resolveLatestVersion(feed.feedId, feed.projectId, packageId);
             tl.debug("Resolved version to: " + version);
+            console.log(tl.loc("Info_ResolveLatestVersion",version));
```
Comment (jmyersmsft) [inline Tasks/DownloadPackageV1/main.ts:77]: Suggestion: move this down two lines (so it prints unconditionally) and make it "Downloading {packageName} {packageVersion}" (or similar)

File: Tasks/DownloadPackageV1/task.json [hunk 2]
```diff
@@ -186,6 +186,7 @@
         "Info_ResolvedToolFromCache": "Resolved from tool cache: %s",
         "Info_DownloadingArtifactTool": "Downloading artifact tool from %s",
         "Info_Downloading": "Downloading package: %s, version: %s using feed id: %s, project: %s",
-        "Info_UsingArtifactToolDownload": "Using artifact tool to download the package"
+        "Info_UsingArtifactToolDownload": "Using artifact tool to download the package",
+        "Info_ResolveLatestVersion": "Resoved latest version to: %s"
```
Comment (jmyersmsft) [inline Tasks/DownloadPackageV1/task.json:190]:
```suggestion
        "Info_ResolveLatestVersion": "Resolved latest version to: %s"
```

**Comment by jmyersmsft** [comment]
Don't forget to bump the task version

File: Tasks/DownloadPackageV1/task.json (hunk 2 above)
Comment (embetten) [reply Tasks/DownloadPackageV1/task.json:190]: good catch

File: Tasks/DownloadPackageV1/main.ts (hunk 1 above)
Comment (embetten) [reply Tasks/DownloadPackageV1/main.ts:77]: we dont have the package name here - only the id. 
I initially tried this:
ex
image
 
but thought it was not super readable.

Comment (jmyersmsft) [reply Tasks/DownloadPackageV1/main.ts:77]: oh, gross. never mind then.

File: make.js [hunk 3]
```diff
@@ -331,7 +331,7 @@ CLI.test = function(/** @type {{ suite: string; node: string; task: string }} */
     rm('-Rf', buildTestsPath);
     mkdir('-p', path.join(buildTestsPath));
     cd(testsPath);
-    run(`tsc --rootDir ${testsPath} --outDir ${buildTestsPath}`);
+    //run(`tsc --rootDir ${testsPath} --outDir ${buildTestsPath}`);
```
Comment (magleaso) [inline make.js:334]: Why?

**Review by magleaso — COMMENTED** [review]
Lgtm other than the comment change, not sure if that was for testing or what

File: make.js (hunk 3 above)
Comment (jmyersmsft) [inline make.js:334]: Is this intentional?

Comment (embetten) [reply make.js]: ah local testing change - reverted

Comment (embetten) [reply make.js]: reverted sorry yall

**Review by jmyersmsft — APPROVED** [review]

**Review by magleaso — APPROVED** [review]

**Pull request merged**